lib/javatests/com/google/android/material/slider/SliderConfigTest.java

```diff
@@ -119,6 +119,29 @@
           /* stepValue = */ 10f,
           /* value = */ 21f,
           IllegalStateException.class,
+        },
+        new Object[] {
+          /* valueFrom = */ 0.1f,
+          /* valueTo = */ 10f,
+          /* stepValue = */ 0.1f,
+          /* value = */ 0.1f,
+          null,
+        },
+        new Object[] {
+          /* valueFrom = */ 0.1f,
+          /* valueTo = */ 10f,
+          /* stepValue = */ 0.1f,
+          /* value = */ 0.1999999f, // This emulates a rounding error from 0.2
+          null,
+        },
+        new Object[][] {
```

Comment:
This weird double array is needed because `ImmutableList` only have overloads for a limited number of values then it switches to `vararg` and since we're using arrays here it becomes confused and thinks that the list type is `Object` rather than `Object[]` which causes compile issues. Using a double array works just fine though.